docs/general_usage.md

```diff
@@ -112,3 +112,8 @@ ps -aux | twilio email:send \
   --subject="Current processes" \
   --text="See attachment"
 ```
+
+## Known limitations
+
+* The default timeout value for the Twilio CLI is 30,000ms (.5 minutes).
```

Comment:
_The default timeout value for Twilio API requests in `twilio-cli` is 30 seconds._